apps/test-pololu370-encoder/src/main.cpp

```diff
@@ -0,0 +1,29 @@
+#include "Pololu37D.h"
+
+int main() {
+  // first pin is yellow wire
+  // second pin is white wire
+  constexpr Encoder::Pololu37D::Config config = {PA_5, PA_6, NC, 0, GPIO::QEI::Encoding::X2_ENCODING};
+  Encoder::Pololu37D enc(config);
+
+  Timer theta_timer, theta_dot_timer;
+  float theta     = 0;
+  float theta_dot = 0;
+  while (true) {
+    theta_timer.reset();
+    theta_dot_timer.reset();
+
+    theta_timer.start();
+    enc.getAngleDeg(theta);  // returns true if successful
```

Comment:
If you are going to comment that it returns true if successful you might as well add a check to see if it returns true. This check can also be used to make sure we get the necessary readings and implement handle it in case we miss an encoder reading

Reply:
I noticed that try/catch statements are not allowed, so I included a few cascading if/else statements.